### ISSUE TITLE
Unexpected return value from 2nd listen call on the same topic

### ISSUE THREAD

**Issue by stalniy** [porsager/postgres#154]
I'm writing tests for `listen` functionality and found that if I add the 2nd listener to the topic that already exists in response I get topic name instead of result with metadata (connection pid, etc):

```js
const result = await sql.listen('test', () => {});
console.log(result);
/*
    [
      count: null,
      command: 'LISTEN',
      state: { status: 'I', pid: 223, secret: -1610071437 }
    ]
*/

const result2 = await sql.listen('test', () => {});
console.log(result2); // test
```

I expect both return the same (or similar) result

**Comment by stalniy** [comment]
by the way, types says that it is always a `PendingRequest` (i.e., `interface PendingRequest extends Promise<[] & ResultMeta<null>> { }`)

**Comment by porsager** [comment]
This was fixed in #155
